test/local/routes/utils/signin.js

```diff
@@ -140,12 +140,8 @@ describe('checkEmailAddress', () => {
   })
 
   it('should throw when email does not match primary after normalization', () => {
-    assert.throws(() => checkEmailAddress(accountRecord, 'secondary@test.net'), (err) => {
-      return err.errno === error.ERRNO.LOGIN_WITH_SECONDARY_EMAIL
-    }, 'threw the correct error')
-    assert.throws(() => checkEmailAddress(accountRecord, 'something@else.org'), (err) => {
-      return err.errno === error.ERRNO.LOGIN_WITH_SECONDARY_EMAIL
-    }, 'threw the correct error')
+    assert.throws(() => checkEmailAddress(accountRecord, 'secondary@test.net'), 'Sign in with this email type is not currently supported')
```

Comment:
Oh, this was the other difference between the two, I forgot to mention. Insist can take a predicate as the second argument to `throws`, whereas chai can only take an error type or message.